osi_common.proto

```diff
@@ -154,6 +154,9 @@ message BaseStationary
 
     /// The relative orientation of the landmark w.r.t its parent frame.
     optional Orientation3d orientation = 3;
+    
+    /// The two dimensional (flat) contour of the object. 
```

Comment:
Good point! I will further clarify. 
I would suggest that the first and last are not equal to avoid redundancy.
Furthermore I would suggest counterclockwise definition of the points. 
Then there is the question about the reference frame. We usually had the middle of the rear axle. But as far as I understood it, in OSI the standard frame is the middle of the bounding box.
@nHirsenkorn Is this correct?